README.md

```
@@ -242,4 +269,4 @@ You cannot roll back an upgrade because of changes to the catalog tables; before
 Then, to upgrade the version of spock that you use to manage your replication cluster, you can remove, build, and upgrade the spock extension like you would any other [PostgreSQL extension](https://www.postgresql.org/docs/17/extend-extensions.html#EXTEND-EXTENSIONS-UPDATES).
 
 
-To review the spock license, visit [here](LICENSE.md).
+To review the spock license, visit [here](LICENSE.md).
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)